### ISSUE TITLE
GraphQL error: You are not authorized to assign a user to this elevated group.

### ISSUE THREAD

**Issue by C0Nd3Mnd** [Requarks/wiki#2424]
When I'm trying to assign a user as the main user (the one created during setup who's in the "Administration" group) to the standard administrator group, the following error occurs:

![msedge_rThbfItS9Q](https://user-images.githubusercontent.com/47672946/92889845-df3d0f00-f416-11ea-9945-0108c5ce4358.png)

I created the user via the administration panel.

**To Reproduce**
1. This is my `docker-compose.yml`:

```yaml
version: '3.1'
services:
  wikijs-db:
    image: postgres:11
    container_name: wikijs-db
    restart: unless-stopped
    volumes:
      - ./db-data:/var/lib/postgresql/data
    environment:
      POSTGRES_PASSWORD: wikijs
      POSTGRES_USER: wikijs
      POSTGRES_DB: wikijs
  wikijs-app:
    image: requarks/wiki:2
    container_name: wikijs-app
    restart: unless-stopped
    depends_on:
      - wikijs-db
    environment:
      DB_TYPE: postgres
      DB_HOST: wikijs-db
      DB_PORT: 5432
      DB_USER: wikijs
      DB_PASS: wikijs
      DB_NAME: wikijs
    ports:
      - 7600:3000
```

2. Setup the wiki.
3. Enter administration and create a user.
4. Open the "Administrators" default group and try to assign a user to it:

![xidxeK78Ho](https://user-images.githubusercontent.com/47672946/92890092-17dce880-f417-11ea-9d41-8212008edcda.png)

After that, the error occurs.

**Expected behavior**
I can add the user to the group with no issue, since I am an administrator and should have all permissions.

**Screenshots**

**Host Info (please complete the following information):**
 - OS: Docker (on Windows)
 - Wiki.js version: 2.5.132
 - Database engine: postgres 11

**Additional context**

**Comment by glogrus** [comment]
I have the same error.

**Comment by NGPixel** [comment]
Fixed by 02c3c660846f6c4f720854dec4c3864583f0aef2
